src/main/java/com/hierynomus/spnego/NegTokenInit.java

```diff
@@ -147,7 +152,7 @@ protected void parseTagged(ASN1TaggedObject asn1TaggedObject) throws SpnegoExcep
                 // Ignore mechListMIC for now...
                 break;
             default:
-                throw new SpnegoException("Unknown Object Tag " + asn1TaggedObject.getTagNo() + " encountered.");
+                logger.warn("Unknown Object Tag {}/{} encountered.. Ignoring", asn1TaggedObject.getTagNo(), asn1TaggedObject.getObject());
```

Comment:
Why is this? The spec is pretty clear as to what this message should and should not contain. Also see the class level comment.

Reply:
Please see the attached trace, the isilon device sends the hostname in the hintName, whereas the spec says it should only contain, "not_defined_in_RFC4178@please_ignore".

![image](https://user-images.githubusercontent.com/428882/55295210-561b4f00-53bf-11e9-8135-1f96731d4e16.png)

Reply:
I've fixed this in a different way now, see the current master. The response token with the negHints is an NegTokenInit2 token, which I totally missed in the initial implementation. Any server initiated NegTokenInit is always a NegTokenInit2 according to MS-SPNG.

Please remove this change from the PR.

Reply:
Thanks, removed my change now and added a test.